tests/load/pipeline/test_refresh_modes.py

```diff
@@ -585,9 +588,7 @@ def test_refresh_staging_dataset(destination_config: DestinationTestConfiguratio
 
 @pytest.mark.parametrize(
     "destination_config",
-    destinations_configs(
-        default_sql_configs=True, default_staging_configs=True, all_buckets_filesystem_configs=True
-    ),
+    destinations_configs(default_sql_configs=True, all_buckets_filesystem_configs=True),
     ids=lambda x: x.name,
 )
 @pytest.mark.parametrize("refresh", ["drop_source", "drop_resource", "drop_data"])
```

Comment:
judging from the explanation of this test, it seems like we do not need it at all but have to test our internal interfaced downstream better (or maybe we don't trust those tests at all? :) )